go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
 	github.com/google/uuid v1.1.1
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.0.0
```

Comment:
This seems like something we certainly should update, but can we do this in a separate PR instead of slipping it in here?

Reply:
It is updated by default when I tried to pull latest kyaml master. Is there any problem due to this ?

Reply:
Ah, it should be fine.